src/Servers/HttpSys/src/RequestProcessing/ResponseBody.cs

```diff
@@ -599,10 +599,7 @@ internal async Task SendFileAsync(string fileName, long offset, long? count, Can
         // It's too expensive to validate the file attributes before opening the file. Open the file and then check the lengths.
         // This all happens inside of ResponseStreamAsyncResult.
         // TODO: Verbose log parameters
-        if (string.IsNullOrWhiteSpace(fileName))
```

Comment:
We are losing the whitespace check, is that ok? A couple other spots have this too.

Reply:
Levi advised against ever using a WhiteSpace check because it does way more than you'd think.